src/components/BMDashboard/Equipment/Update/UpdateEquipment.jsx

```diff
@@ -0,0 +1,354 @@
+                        'condition',
+                      )
+                    : 'unknown'
+                }
+                className="read-only-input"
+                id="itemStatus"
+              />
+            </Col>
+            <Col md={4}>
+              <Label for="itemOwnership">Ownership</Label>
+              <Input
+                readOnly
+                value={safeData(equipmentDetails, 'purchaseStatus')}
+                className="read-only-input"
+                id="itemOwnership"
+              />
+            </Col>
+          </Row>
+          {equipmentDetails && equipmentDetails.purchaseStatus === 'Rental' && (
+            <Row form>
+              <Col md={4}>
+                <Label for="rentalEndDate">Rental End Date</Label>
+                <Input
+                  readOnly
+                  value={
+                    equipmentDetails
+                      ? safeData(equipmentDetails, 'rentalDueDate').split('T')[0]
+                      : 'unknown'
+                  }
+                  className="read-only-input"
+                  id="rentalEndDate"
+                />
+              </Col>
+              <Col md={4}>
+                <Label for="daysLeft">Days Left</Label>
+                <Input
+                  readOnly
+                  value={calculateDaysLeft(equipmentDetails?.rentalDueDate)}
+                  className="read-only-input"
+                  id="daysLeft"
+                />
+              </Col>
+            </Row>
+          )}
+        </FormGroup>
+        <Row form>
+          <Col md={12}>
+            <div className="update-confirm-text">
+              Please confirm you are updating the status of the tool or equipment shown above.
+            </div>
+          </Col>
+        </Row>
+        <Row form>
+          <Col md={4}>
+            <FormGroup>
+              <Label for="updateDate">Update Date</Label>
+              <Input
+                type="date"
+                name="updateDate"
+                id="updateDate"
+                value={updateDate}
+                onChange={e => setUpdateDate(e.target.value)}
+                placeholder="DD/MM/YYYY"
+                className="form-control"
+              />
+            </FormGroup>
+          </Col>
+        </Row>
+        <Row>
+          <Col md={8}>
+            <FormGroup>
+              <Label for="status">Status/Condition Now</Label>
+              <Input
+                id="status"
+                name="status"
+                type="select"
+                value={status}
+                onChange={e => setStatus(e.target.value)}
+              >
+                <option value="1">Working well</option>
+                <option value="2">Broken/Needs repair</option>
+                <option value="3">Stolen/Lost</option>
+                <option value="4">End of life</option>
+                <option value="5">Returned</option>
+              </Input>
+            </FormGroup>
```

Comment:
If is read only and the user can not edit then I don't believe is user friendly putting in <input> tag. Instead, should make in a div colum grid and rows.

Reply:
Hi Sharif, thank you so much for your constructive feedback! I have fixed the code from your suggestion, now all the read only input is replace by div. Thank you!